src/main/java/io/strimzi/test/container/StrimziKafkaContainer.java

```diff
@@ -116,7 +116,7 @@ protected void doStart() {
             this.imageNameProvider.complete(KafkaVersionService.strimziTestContainerImageName(this.kafkaVersion));
         }
         try {
-            if (this.useKraft && ((this.kafkaVersion != null && this.kafkaVersion.equals("2.8.1")) || this.imageNameProvider.get().contains("2.8.1"))) {
+            if (this.useKraft && ((this.kafkaVersion != null && this.kafkaVersion.equals("2.8.2")) || this.imageNameProvider.get().contains("2.8.2"))) {
```

Comment:
Perhaps we should use `startsWith("2.")`?

Reply:
Yeap, I thought about it but did not want to do it in this PR. But I will do it.